src/frontend/apps/lti_site/components/LocalizedTimedTextTrackUpload/TimedTextTrackItem/index.spec.tsx

```diff
@@ -39,11 +38,19 @@ describe('<TimedTextTrackItem />', () => {
     });
   });
 
-  it('renders the component for a correctly uploaded timed text track', () => {
+  afterEach(() => {
+    fetchMock.restore();
+  });
+
+  it('renders the component for a correctly uploaded timed text track', async () => {
+    fetchMock.mock(
```

Comment:
Same as before, if the same mock is in all the tests, considere to add it in the beforeEach methods.